docs/developing.rst

```diff
@@ -9,7 +9,8 @@ Install the following software packages:
 * Python 3.4 or better available as ``python3`` on the path (verify with: ``python3 --version``)
 * ``pip3`` available on the path (verify with ``pip3 --version``)
 * ``virtualenv`` available on the path (verify with ``virtualenv --version``)
-* JDK 8 or 9
+* Pyenv insatlled and ``eval "$(pyenv init -)"`` is added to the shell configuration file. For more details please refer to https://github.com/pyenv/pyenv.
+* JDK 11
```

Comment:
This will change in the future, so maybe instead point to the Java requirements for building Elasticsearch, depending on the version?

Reply:
Thank you! Referenced the link from @danielmitterdorfer.